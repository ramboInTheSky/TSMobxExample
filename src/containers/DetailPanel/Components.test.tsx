import React from 'react'
import { shallow } from 'enzyme'

import {
	Wrapper,
	Field,
	FieldsContainer,
	ActionsBar,
	Name,
	ButtonsContainer,
	SingleColumnFieldSet,
	InputContainer,
	Input,
	TwoColumnsFieldset,
	NotesInputContainer,
} from './Components'

describe('<Wrapper />', () => {
	it('renders correctly', () => {
		expect(shallow(<Wrapper />)).toMatchSnapshot()
	})
})

describe('<Field />', () => {
	it('renders correctly', () => {
		expect(shallow(<Field />)).toMatchSnapshot()
	})
})

describe('<FieldsContainer />', () => {
	it('renders correctly', () => {
		expect(shallow(<FieldsContainer />)).toMatchSnapshot()
	})
})

describe('<ActionsBar />', () => {
	it('renders correctly', () => {
		expect(shallow(<ActionsBar />)).toMatchSnapshot()
	})
})

describe('<Name />', () => {
	it('renders correctly', () => {
		expect(shallow(<Name />)).toMatchSnapshot()
	})
})

describe('<ButtonsContainer />', () => {
	it('renders correctly', () => {
		expect(shallow(<ButtonsContainer />)).toMatchSnapshot()
	})
})

describe('<SingleColumnFieldSet />', () => {
	it('renders correctly', () => {
		expect(shallow(<SingleColumnFieldSet />)).toMatchSnapshot()
	})
})

describe('<InputContainer />', () => {
	it('renders correctly', () => {
		expect(shallow(<InputContainer />)).toMatchSnapshot()
	})
})

describe('<Input />', () => {
	it('renders correctly', () => {
		expect(shallow(<Input />)).toMatchSnapshot()
	})
})

describe('<TwoColumnsFieldset />', () => {
	it('renders correctly', () => {
		expect(shallow(<TwoColumnsFieldset />)).toMatchSnapshot()
	})
})

describe('<NotesInputContainer />', () => {
	it('renders correctly', () => {
		expect(shallow(<NotesInputContainer />)).toMatchSnapshot()
	})
})
