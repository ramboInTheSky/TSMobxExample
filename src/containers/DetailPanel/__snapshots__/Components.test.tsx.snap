// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActionsBar /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<ButtonsContainer /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<Field /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<FieldsContainer /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<Input /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<InputContainer /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<Name /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<NotesInputContainer /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<SingleColumnFieldSet /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<TwoColumnsFieldset /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<Wrapper /> renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
